jekyll/_cci2/server/Operator/vm-service.adoc

```diff
@@ -35,39 +37,40 @@ There is also an option to change the port used for VM service. The default is `
-* *Subnets* (required): Choose subnets (public or private) where the VMs should be deployed. Note that all subnets must be in the same availability zone.
-* *Security Group ID* (required): This is the security group that will be attached to the VMs.
-+
-The recommended security group configuration can be found in the https://circleci.com/docs/2.0/server-3-install-hardening-your-cluster[Hardening Your Cluster] section.
-* *Number of <VM-type> VMs to keep prescaled*: By default, this field is set to 0, which creates and provisions instances of a resource type on demand. You have the option of preallocating up to five instances per resource type. Preallocating instances lowers the start time, allowing for faster machine and `remote_docker` builds. 
-+
-NOTE: Preallocated instances are always running and could potentially increase costs. Decreasing this number may also take up to 24 hours for changes to take effect. You have the option of terminating those instances manually, if required.
-+
-WARNING: If https://circleci.com/docs/2.0/docker-layer-caching/[Docker Layer Caching (DLC)] is used, VM Service instances need to be spun up on demand. For this to happen, **either** check any preallocated instances are in use, **or** set both remote Docker and `machine` preallocated instance fields to `0`.
-+
-NOTE: When using preallocated instances, be aware that a cron job is scheduled to cycle through these instances once per day to ensure they do not end up in an unworkable state.
+You will need to add a section to your values.yaml file to configure your VM Service to work with AWS EC2. 
 
 ==== Authentication
 One of the following options is required. Either select "IAM Keys" and provide:
 
 * *Access Key ID* (required): https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html[Access Key ID] for EC2 access.
-+
+* *Secret Key* (required): https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html[Secret Key] for EC2 access.
 NOTE: The Access Key and Secret
 Key used by VM Service differs from the policy used by object storage in the previous section.
-* *Secret Key* (required): https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html[Secret Key] for EC2 access.
 
 Or select "IAM role" and provide:
 
 * *Role ARN* (required): 
 https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html[Role ARN for Service Accounts] (Amazon Resource Name) for EC2 access.
 
+```
+vm_service:
+  providers:
+    ec2:
+      enabled: true
+      region: <region>
+      # Subnets must be in the same availability zone
+      subnets:
+      - <subnet-id>
+      securityGroupId: <security-group-id>
+
+      # Authenticate with IAM access keys
+      accessKey: <access-key>
+      secretKey: <secret-key>
+      # or IRSA (IAM roles for service accounts)
+      irsaRole: <role-arn>
+
```

Comment:
I think we should include windowsAMI, linuxAMI, and tags here, as well.

Reply:
My thought was to keep these instructions limited to a very "standard" example.  The values.yaml file itself can be the source of more advanced configuration options.